recursio-client/UI/Menus/StartMenu.gd

```diff
@@ -135,15 +146,29 @@ func _on_play_tutorial() -> void:
 
 
 func _on_play_online() -> void:
+	_btn_cancel_online.show()
 	_toggle_enabled_start_menu_buttons(false)
 	Server.connect_to_server(REMOTE_SERVER_IP)
 
 
+func _on_cancel_online() -> void:
+	_btn_cancel_online.hide()
+	_toggle_enabled_start_menu_buttons(true)
+	Server.disconnect_from_server()
+
+
 func _on_play_local() -> void:
+	_btn_cancel_local.show()
 	_toggle_enabled_start_menu_buttons(false)
 	Server.connect_to_server(_lineEdit_local_ip.text)
 
 
+func _on_cancel_local() -> void:
```

Comment:
Can't you just make one `on_connect_cancel `function that takes the button as an arg. As far as I can tell `_on_cancel_online `and on_connect_cancel do pretty much the same.